api/types/types.go

```diff
@@ -105,6 +105,7 @@ type Ping struct {
 	APIVersion   string
 	OSType       string
 	Experimental bool
+	Buildkit     bool
```

Comment:
`BuildKit` that's how it's spelled elsewhere especially for API types

Reply:
Should it be `FeatureBuildKit` ? Also, need to make the flag longer, or modify the config condition.